<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>V2RAYQT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_resource.qrc">
    <normaloff>:/MAIN_LOGO</normaloff>:/MAIN_LOGO</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
      <item>
       <widget class="QTableView" name="server_table_view"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_log"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_server">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="action_add_server"/>
    <addaction name="action_add_files"/>
    <addaction name="action_remove_server"/>
    <addaction name="action_modify_server"/>
    <addaction name="separator"/>
    <addaction name="action_import_vmess_link"/>
    <addaction name="action_export_vmess_link"/>
    <addaction name="separator"/>
    <addaction name="action_update_subscription"/>
    <addaction name="action_clear_subscription"/>
   </widget>
   <widget class="QMenu" name="menuSeting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="action_select_template"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="action_restart_process"/>
    <addaction name="action_terminate_process"/>
   </widget>
   <addaction name="menu_server"/>
   <addaction name="menuSeting"/>
   <addaction name="menuProcess"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_add_server">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="action_remove_server">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="action_modify_server">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
  <action name="action_import_vmess_link">
   <property name="text">
    <string>Import link from the clipboard</string>
   </property>
  </action>
  <action name="action_export_vmess_link">
   <property name="text">
    <string>Export link to clipboard</string>
   </property>
  </action>
  <action name="action_update_subscription">
   <property name="text">
    <string>Update subscription</string>
   </property>
  </action>
  <action name="action_clear_subscription">
   <property name="text">
    <string>Clear subscription</string>
   </property>
  </action>
  <action name="action_select_template">
   <property name="text">
    <string>Select template</string>
   </property>
  </action>
  <action name="action_add_files">
   <property name="text">
    <string>Add files</string>
   </property>
  </action>
  <action name="action_restart_process">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="action_terminate_process">
   <property name="text">
    <string>Terminate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui_resource.qrc"/>
 </resources>
 <connections/>
</ui>
